mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -540,6 +549,25 @@ func resourceComputeInstance() *schema.Resource {
 				},
 			},
 
+	                "params": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `Stores additional params passed with the request, but not persisted as part of resource payload.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"resource_manager_tags": {
+							Type:         schema.TypeMap,
+							Optional:     true,
```

Comment:
Should we consider making the field required? Would a user every want to specify `params {}`?

Reply:
I think we should leave it optional, because params will have some more fields to be exposed as optional.